src/main/java/org/broadinstitute/consent/http/resources/DacResource.java

```diff
@@ -192,9 +193,10 @@ public Response removeDacChair(@Auth AuthUser authUser, @PathParam("dacId") Inte
     @Produces("application/json")
     @RolesAllowed({ADMIN, MEMBER, CHAIRPERSON})
     public Response findAllDacDatasets(@Auth AuthUser user, @PathParam("dacId") Integer dacId) {
-        findDacById(dacId);
-        Set<DatasetDTO> datasets = dacService.findDatasetsByDacId(user, dacId);
-        return Response.ok().entity(unmarshal(datasets)).build();
+        Dac dac = findDacById(dacId);
+        checkUserRoleInDac(dac, user);
+        List<Dataset> datasets = dacService.findDatasetsByDacId(dacId);
+        return Response.ok().entity(datasets).build();
```

Comment:
Line 9 has an unused import, `import java.util.Set;`

Reply:
And line 31 has one too: `import org.broadinstitute.consent.http.models.dto.DatasetDTO;`